<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FacturasForm</class>
 <widget class="QWidget" name="FacturasForm">
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="formLayout">
     <item row="0" column="0"><widget class="QLabel" name="labelID"><property name="text"><string>ID</string></property></widget></item>
     <item row="0" column="1"><widget class="QLineEdit" name="inputID"/></item>
     <item row="1" column="0"><widget class="QLabel" name="labelFechaEmision"><property name="text"><string>Fecha Emisión</string></property></widget></item>
     <item row="1" column="1"><widget class="QLineEdit" name="inputFechaEmision"/></item>
     <item row="2" column="0"><widget class="QLabel" name="labelFechaVencimiento"><property name="text"><string>Fecha Vencimiento</string></property></widget></item>
     <item row="2" column="1"><widget class="QLineEdit" name="inputFechaVencimiento"/></item>
     <item row="3" column="0"><widget class="QLabel" name="labelNombreCliente"><property name="text"><string>Nombre Cliente</string></property></widget></item>
     <item row="3" column="1"><widget class="QLineEdit" name="inputNombreCliente"/></item>
     <item row="4" column="0"><widget class="QLabel" name="labelTotalNeto"><property name="text"><string>Total Neto</string></property></widget></item>
     <item row="4" column="1"><widget class="QLineEdit" name="inputTotalNeto"/></item>
     <item row="5" column="0"><widget class="QLabel" name="labelTotalIVA"><property name="text"><string>Total IVA</string></property></widget></item>
     <item row="5" column="1"><widget class="QLineEdit" name="inputTotalIVA"/></item>
     <item row="6" column="0"><widget class="QLabel" name="labelTotal"><property name="text"><string>Total</string></property></widget></item>
     <item row="6" column="1"><widget class="QLineEdit" name="inputTotal"/></item>
     <item row="7" column="0"><widget class="QLabel" name="labelIngreso"><property name="text"><string>Ingreso</string></property></widget></item>
     <item row="7" column="1"><widget class="QLineEdit" name="inputIngreso"/></item>
     <item row="8" column="0"><widget class="QLabel" name="labelClasificacion"><property name="text"><string>Clasificación</string></property></widget></item>
     <item row="8" column="1"><widget class="QLineEdit" name="inputClasificacion"/></item>
     <item row="9" column="0"><widget class="QLabel" name="labelAnio"><property name="text"><string>Año</string></property></widget></item>
     <item row="9" column="1"><widget class="QLineEdit" name="inputAnio"/></item>
     <item row="10" column="0"><widget class="QLabel" name="labelEstado"><property name="text"><string>Estado</string></property></widget></item>
     <item row="10" column="1">
      <widget class="QComboBox" name="inputEstado">
       <item><property name="text"><string>Cancelado</string></property></item>
       <item><property name="text"><string>Adeuda</string></property></item>
      </widget>
     </item>
     <item row="11" column="0"><widget class="QLabel" name="labelRutCliente"><property name="text"><string>RUT Cliente</string></property></widget></item>
     <item row="11" column="1"><widget class="QLineEdit" name="inputRutCliente"/></item>
     <item row="12" column="0"><widget class="QLabel" name="labelBuscar"><property name="text"><string>Buscar</string></property></widget></item>
     <item row="12" column="1"><widget class="QLineEdit" name="inputBuscar"/></item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="botonesLayout">
     <item><widget class="QPushButton" name="btnAgregar"><property name="text"><string>Agregar</string></property></widget></item>
     <item><widget class="QPushButton" name="btnEditar"><property name="text"><string>Editar</string></property></widget></item>
     <item><widget class="QPushButton" name="btnEliminar"><property name="text"><string>Eliminar</string></property></widget></item>
     <item><widget class="QPushButton" name="btnBuscar"><property name="text"><string>Buscar</string></property></widget></item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tablaFacturas"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
